sonar-java-plugin/src/main/resources/org/sonar/l10n/java/rules/java/S7435.json

```diff
@@ -0,0 +1,34 @@
+{
+  "title": "Processing persistent unique identifiers is security-sensitive",
+  "type": "SECURITY_HOTSPOT",
+  "status": "ready",
+  "remediation": {
+    "func": "Constant\/Issue",
+    "constantCost": "5min"
+  },
+  "tags": [
+    "android"
+  ],
+  "defaultSeverity": "Minor",
+  "ruleSpecification": "RSPEC-7435",
+  "sqKey": "S7435",
+  "scope": "All",
+  "securityStandards": {
+    "ASVS 4.0": [
+      "8.3.3"
+    ],
+    "OWASP Mobile Top 10 2024": [
+      "M6"
+    ],
+    "CWE": [
+      213
+    ]
+  },
+  "quickfix": "infeasible",
```

Comment:
I have set the quickfix to infeasible because, while we could just remove the call, the resulting code would very likely be non-compiling and possibly functionally wrong: the user would be using these identifiers to build composite identifiers, intents extra, or the like.
It is not possible to determine what is the right replacement for the telephony/subscription/ads identifier used - it depends on the specific scenario.